### PR TITLE
🥓🐣 ↝ Different models are loaded in based on their respective anomaly…

### PR DESCRIPTION
**Pull request by Gizmotronn**
…'s temperature

Essentially, as of right now we just have one model being shown. I want to customise more things like the colours, biome textures, etc via the API, but for now I’m just going to be affecting the trees based on the API, and the other values & models will be rendered in randomly. Some of the models won’t have trees so we’ll have to choose something else to modify with the value returned from the current iteration of the flask api in signal-k/sytizen but for now we’ll just leave that blank as well.

In short → a method to render different Unity builds. Let’s add it as an additional field in the Supabase table, rather than doing any real interactions with unity for now.

This method should suffice for now.

### CODE REVIEW

**Review by Gizmotronn — COMMENTED** [review]
Builds correctly :)
